ros/src/twist_controller/dbw_node.py

```diff
@@ -69,9 +69,9 @@ def __init__(self):
         # Logging data in csv file
         self.log_to_csv = True
         if self.log_to_csv:
-            self.log_handle = self.log_init('CarND_performance.csv')
+            self.log_handle = self.log_init('dbw_node.csv')
 
-        self.time_init = rospy.get_rostime()
+        self.time_init = rospy.get_time()
```

Comment:
both get_time and get_rostime are used, which is inconsistent. I would just use one of them in the file, ideally the one with best precision since we run at a high rate (I think get_rostime is more accurate)

Reply:
I have reverted this change